src/app/maintenance-page/maintenance-page.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { MaintenancePageComponent } from './maintenance-page.component';
+
+describe('MaintenancePageComponent', () => {
+  let component: MaintenancePageComponent;
+  let fixture: ComponentFixture<MaintenancePageComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ MaintenancePageComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(MaintenancePageComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Please provide a test that checks if cogs are spinning in the correct direction.